### PR TITLE
SPHNX-2147-Funding-Block-Explore-config

### PR DESCRIPTION
**Pull request by Mathieu-Gilli**
[SPHNX-2147](https://alayacare.atlassian.net/browse/SPHNX-2147)

### CODE REVIEW

File: looker_config.json [hunk 1]
```diff
@@ -100,6 +100,10 @@
       "label": "Employee Retention",
       "view": "retention_employee_tier_5",
       "release_flag": "LAB-1336-employee-retention-explore"
+    },
+    "funding_block": {
```
Comment (syedpriom) [inline looker_config.json:104]: @Mathieu-Gilli 

1. I think we'll need to update this file as well: https://github.com/AlayaCare/api.reporting/blob/develop/alaya/api/reporting/looker/helpers.py#L220 hosted on S3.

2. Is it possible to rename `funding_block` to `fundingblock`? Currently for explore permissions the value we're mapping to is `fundingblock`.

Comment (tix0) [reply looker_config.json:104]: For 1. There is an action item on this repo, once you merge the file on s3 will be updated.

Comment (Mathieu-Gilli) [reply looker_config.json:104]: thanks, I'll action on item #2

Comment (Mathieu-Gilli) [reply looker_config.json:104]: Instead for item 2 we updated mapping here https://github.com/AlayaCare/api.reporting/pull/666/files

**Review by puneetkaursidhu — APPROVED** [review]

**Pull request closed without merging**